mux/src/window.rs

```diff
@@ -131,6 +131,8 @@ impl Window {
 
         if len > 0 && self.active >= len {
             self.set_active_without_saving(len - 1);
+        } else if len > 0 {
+            self.set_active_without_saving(self.active);
```

Comment:
Seems simpler to invalidate in the else branch?
 
```suggestion
        } else {
            self.invalidate();
```

Reply:
I thought call `set_active_without_saving` would be better, because it's exactly what we need - set a new active tab which index = prev active tab index.